conf/machine/include/imx-base.inc

```diff
@@ -385,10 +385,9 @@ MACHINE_FIRMWARE:append:mx6sll-generic-bsp = " firmware-imx-epdc"
 MACHINE_FIRMWARE:append:mx6ull-generic-bsp = " firmware-imx-epdc"
 MACHINE_FIRMWARE:append:mx53-generic-bsp   = " firmware-imx-vpu-imx53 firmware-imx-sdma-imx53"
 MACHINE_FIRMWARE:append:mx51-generic-bsp   = " firmware-imx-vpu-imx51 firmware-imx-sdma-imx51"
-MACHINE_FIRMWARE:append:mx8mm-generic-bsp  = " linux-firmware-imx-sdma-imx7d"
-MACHINE_FIRMWARE:append:mx8mn-generic-bsp  = " linux-firmware-imx-sdma-imx7d firmware-imx-easrc-imx8mn"
-MACHINE_FIRMWARE:append:mx8mp-generic-bsp  = " linux-firmware-imx-sdma-imx7d firmware-imx-easrc-imx8mn firmware-imx-xcvr-imx8mp firmware-sof-imx"
-MACHINE_FIRMWARE:append:mx8mq-generic-bsp  = " linux-firmware-imx-sdma-imx7d"
+MACHINE_FIRMWARE:append:mx8-generic-bsp    = " linux-firmware-imx-sdma-imx7d"
+MACHINE_FIRMWARE:append:mx8mn-generic-bsp  = " firmware-imx-easrc-imx8mn"
+MACHINE_FIRMWARE:append:mx8mp-generic-bsp  = " firmware-imx-easrc-imx8mn firmware-imx-xcvr-imx8mp firmware-sof-imx"
```

Comment:
Considering future maintenance, duplicating the firmware files is easier for review and makes the core more obvious. 

Could you remove this rework?